src/main/java/minegame159/meteorclient/utils/misc/Timer.java

```diff
@@ -0,0 +1,46 @@
+package minegame159.meteorclient.utils.misc;
+
+public class Timer {
```

Comment:
Please name the methods more descriptively, I have no idea what some of them mean.